datum/schema/org/openstax/ec/nudged/v1/nudged_swagger.rb

```diff
@@ -0,0 +1,59 @@
+class Ec::Nudged::V1::NudgedSwagger
+  include Swagger::Blocks
+  include OpenStax::Swagger::SwaggerBlocksExtensions
+
+  swagger_schema :Nudged do
```

Comment:
See above binding comment, need some way to get the "v1" into the binding class name so that we can later have a "v2" that is different.